src/Coverage/CoverageCodeLensProvider.ts

```diff
@@ -0,0 +1,47 @@
+import * as vscode from 'vscode'
+
+import { JestExt } from '../JestExt'
+import { extensionName } from '../appGlobals'
+
+const coverageCommand = `${extensionName}.coverage.metrics`
+
+export function registerCoverageCodeLens(jestExt: JestExt) {
+  return [
+    vscode.languages.registerCodeLensProvider(
+      { pattern: '**/*.{ts,tsx,js,jsx}' },
+      new CoverageCodeLensProvider(jestExt)
+    ),
+    vscode.commands.registerCommand(coverageCommand, () => {
+      vscode.window.showInformationMessage('You have unit tests!')
+    }),

```

Comment:
Were we planning to use this command for anything? It looks like we could remove it now.

Reply:
At one point I had a plan for it.  Not clue what it was now though :)  I think I was planning on maybe using it to toggle the coverage overlay (ended up doing that quick/dirty later though).

Reply:
Oh, I know that feeling. I'll pop that out now.